features/src/main/feature/feature.xml

```diff
@@ -29,13 +29,14 @@
         <feature>openhab-core-model-item</feature>
         <feature>openhab-core-model-sitemap</feature>
         <feature>openhab-core-ui-icon</feature>
-        <feature>openhab-core-ui</feature>
+	    <feature>openhab-core-ui</feature>
         <feature dependency="true">openhab.tp-jaxb</feature>
+        <feature dependency="true">openhab.tp-jackson</feature>
```

Comment:
Where do you actually require Jackson?

Reply:
The generated files from the openapi.yaml have jackson imports. But as I just saw: those imports aren't even used. I will try to find an option in the generator to get rid of th unused imports.

Reply:
Ok forget that, they so use jackson annotations, so we need jackson here.

Reply:
So you say the annotations are actively used at runtime by the generated code? If that's the case, we indeed have to keep it.